### PR TITLE
Updated linux installation instructions to recommend snap, added snap installation instructions

### PR DESCRIPTION
**Pull request by ekingery**
Related to [issue #700](https://github.com/exercism/cli/issues/700) and [PR #886](https://github.com/exercism/cli/pull/886).

### CODE REVIEW

File: contents/linux/introduction.tw2 [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
+You can use the [snap package](https://snapcraft.io/exercism) to install the command-line client, or install it manually. If you are unsure which one to choose, try using snap first. More information is available in the next section.
 
 ---
 
-## Do you know your process architecture (x86 (32-bit) vs x86_64 (64-bit) vs ARM)?
+## I would like to install the CLI
 
-- [[Yes->Linux tarball]]
-- [[No->Linux architecture]]
+- [[Using the snap->Linux - Snap]]
```
Comment (kntsoriano) [inline contents/linux/introduction.tw2]:
```suggestion
- [[Using the snap package->Linux - Snap]]
```

File: contents/linux/snap.tw2 [hunk 2]
```diff
@@ -0,0 +1,25 @@
+::Linux - Snap
+# Installing Exercism on Linux using Snap
+
+To install BINARY_NAME via the snap package manager, run:
```
Comment (kntsoriano) [inline contents/linux/snap.tw2]:
```suggestion
To install `BINARY_NAME` via the snap package manager, run:
```

**Comment by kntsoriano** [comment]
Maybe we should add a link to Install via snap in the Linux tarball page when it becomes too complicated for the user.

**Comment by ekingery** [comment]
> Maybe we should add a link to Install via snap in the Linux tarball page when it becomes too complicated for the user.

For sure, good suggestion and thanks for the review! I'll add this, rebase, and we should be all set.

**Comment by ekingery** [comment]
> > Maybe we should add a link to Install via snap in the Linux tarball page when it becomes too complicated for the user.
> 
> For sure, good suggestion and thanks for the review! I'll add this, rebase, and we should be all set.

This is done, we're ready for merge. Given that the snap is already published, we can merge this any time, it is independent of [issue #700](https://github.com/exercism/cli/issues/700) and [PR #886](https://github.com/exercism/cli/pull/886).

**Comment by rpottsoh** [comment]
@kntsoriano would you be able to build the walkthrough and update the appropriate files in website-copy?  Index.html @ website-copy

I acknowledge that the README describes how to do this.  I am not in a position at the moment to perform this task myself.

**Comment by kntsoriano** [comment]
@rpottsoh I've opened https://github.com/exercism/website-copy/pull/1348 to merge this in. There's a couple of comments there which I will resolve today.

**Pull request merged**